test/myapp18_web/user_auth_test.exs

```diff
@@ -152,6 +156,52 @@ defmodule Myapp18Web.UserAuthTest do
       refute get_session(conn, :user_token)
       refute conn.assigns.current_scope
     end
+
+    test "does not reissue the session token after only a few hours", %{conn: conn, user: user} do
```

Comment:
Let's nuke this test and instead assert in one of the existing tests that the session token is the same before and after.